flow/Makefile

```diff
@@ -401,18 +401,16 @@ ifeq ($(SYNTH_HIERARCHICAL), 1)
 $(RESULTS_DIR)/1_1_yosys.v: $(SYNTH_STOP_MODULE_SCRIPT)
 endif
 
-$(RESULTS_DIR)/1_1_yosys.v: $(DONT_USE_LIBS) $(WRAPPED_LIBS) $(DONT_USE_SC_LIB) $(DFF_LIB_FILE) $(VERILOG_FILES) $(CACHED_NETLIST) $(LATCH_MAP_FILE) $(ADDER_MAP_FILE)
+$(RESULTS_DIR)/1_1_yosys.v $(RESULTS_DIR)/1_synth.sdc &: $(DONT_USE_LIBS) $(WRAPPED_LIBS) $(DONT_USE_SC_LIB) $(DFF_LIB_FILE) $(VERILOG_FILES) $(CACHED_NETLIST) $(LATCH_MAP_FILE) $(ADDER_MAP_FILE)
 	mkdir -p $(RESULTS_DIR) $(LOG_DIR) $(REPORTS_DIR)
 	($(TIME_CMD) $(YOSYS_CMD) $(YOSYS_FLAGS) -c $(SYNTH_SCRIPT)) 2>&1 | tee $(LOG_DIR)/1_1_yosys.log
+	cp $(SDC_FILE) $(RESULTS_DIR)/1_synth.sdc
+	($(TIME_CMD) $(OPENROAD_CMD) $(SCRIPTS_DIR)/synth_metrics.tcl) 2>&1 | tee -a $(LOG_DIR)/1_1_yosys.log
```

Comment:
This will add a bunch of runtime for a large design and I'm not sure it will generally be useful.  Perhaps it should be its own target for a specific workflow.

Reply:
I don't see any report at the beginning of the floorplan.  In any case you would still have to load the design twice.

Reply:
> I don't see any report at the beginning of the floorplan. In any case you would still have to load the design twice.

Ah, towards the end of floorplan:


https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/0f70f74c26b9f697132611f537844f1f6271ded5/flow/scripts/floorplan.tcl#L115

Reply:
How about I move it into the synth target, but not to synthesis?